scripts/windows-build.ps1

```diff
@@ -0,0 +1,19 @@
+$ErrorActionPreference = "Stop"
```

Comment:
Please add yourself as the copyright owner, see the other SPDX headers headers for reference. The license is up to you, but probably CC0-1.0, MIT or some other permissive license other than GPL is a good idea.